Backend/src/test/java/com/senla/controller/mock/AuthControllerTest.java

```diff
@@ -0,0 +1,75 @@
+package com.senla.controller.mock;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.senla.dto.user.DtoCreateUser;
+import com.senla.dto.user.ForgotPasswordDto;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.*;
+
+class AuthControllerTest extends AbstractMockControllerTest {
+
+    @Autowired private ObjectMapper objectMapper;
+
+    @Test
+    void givenUser_whenRegister_thenReturnUser() throws Exception {
+        DtoCreateUser dtoCreateUser =
+                DtoCreateUser.builder()
+                        .email("email@gmail.com")
+                        .password("password")
+                        .matchingPassword("password")
+                        .build();
+        mockMvc.perform(
+                        post("/api/auth/registration")
+                                .content(objectMapper.writeValueAsString(dtoCreateUser))
+                                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
```

Comment:
Usually  ```java .andExpect(status().isOk())``` is enough with MockMvc

If you want check specific values I would something like that
```java
        MvcResult mvcResult = mockMvc.perform(
                        post("/api/auth/registration")
                                .content(objectMapper.writeValueAsString(dtoCreateUser))
                                .contentType(MediaType.APPLICATION_JSON))
                .andExpect(status().isOk()).andReturn();

        DtoUser returnedUser = objectMapper.readValue(mvcResult.getResponse().getContentAsString(), DtoUser.class);
        Assertions.assertEquals(dtoCreateUser.getEmail(), returnedUser.getEmail());
```